code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -103,6 +103,9 @@
 				"You unbuckle yourself from \the [src].",
 				"You hear metal clanking.")
 		playsound(src, 'sound/misc/buckle_unclick.ogg', 50, 1)
+		if(M.disabilities & ANEMIA)
+			M.sleeping += 2
```

Comment:
from this code it seems like you can force someone to sleep for a long time by rapidly unbuckling and rebuckling them, since the sleep time stacks on top of eachother

Reply:
that is hilarious and i will fix it